examples/partial_execution/README.md

```diff
@@ -0,0 +1,34 @@
+# Partial execution
+
+The structure of the example is visualized in [pipeline.svg](./pipeline.svg).
+
+When run without arguments the example starts a `service`, waits for its
+availability and then executes a number of tests which are sequenced but
+independent.
+
+To skip `test_1` and `test_0` (see
+[partial_execution_0.svg](./partial_execution_0.svg)) the following command
```

Comment:
Also include the image, might want to change that in other examples too, maybe even the front README.md